tests/test-cases/Group1-Docker-Commands/1-06-Docker-Run.robot

```diff
@@ -239,5 +239,7 @@ Docker run --rm concurrent
     \   Log  ${res.stdout}
     \   Should Be Equal As Integers  ${res.rc}  0
 
+    Sleep  3 minutes
```

Comment:
we could instead use a docker event stream for deletions with a timeout so we don't wait longer than necessary. That said, not suggesting changing it in this PR.